src/typeDefs/inputs.ts

```diff
@@ -285,6 +285,12 @@ export const inputs = gql`
     id: ID
   }
 
+  input UpdateOrgUserTypeInput {
+    userType: String
+    id: ID
+    organizationId: ID
```

Comment:
Make all these arguments required, else it doesn't make to trigger the mutation on client side